client/pluginManger.go

```diff
@@ -133,6 +139,26 @@ func (p *PluginManager) InstallPlugin(names []string) (err error) {
 	return
 }
 
+// InstallPluginWithVersion install a plugin by name & version
+func (p *PluginManager) installPluginWithVersion(name string) (err error) {
+	pluginAPI := PluginAPI{}
+	pluginName := "%s.hpi"
+	pluginVersion := strings.Split(name, "@")
+	url := fmt.Sprintf("http://updates.jenkins-ci.org/download/plugins/%s/%s/%s.hpi",
+		pluginVersion[0], pluginVersion[1], pluginVersion[0])
+
+	statusCode := pluginAPI.download(url, name)
```

Comment:
You can delete it by the below way:
`defer os.Remove(fmt.Sprintf(pluginName, name))`